src/pybind/cephfs/cephfs.pyx

```diff
@@ -143,8 +143,9 @@ cdef extern from "cephfs/libcephfs.h" nogil:
     int ceph_sync_fs(ceph_mount_info *cmount)
     int ceph_fsync(ceph_mount_info *cmount, int fd, int syncdataonly)
     int ceph_conf_parse_argv(ceph_mount_info *cmount, int argc, const char **argv)
+    int ceph_chmod(ceph_mount_info *cmount, const char *path, mode_t mode)
```

Comment:
you `chmod`ed cephfs.pyx to 755. could you revert this change?

Reply:
you mean the cephfs.pyx file permission right? I will do it.

Reply:
yup, the permission bits.

Reply:
@rpavani1998 #23417